### ISSUE TITLE
Daikin integration not found

### ISSUE THREAD

**Issue by genga91** [rospogrigio/daikin_residential#26]
Hi Rospogrigio,
I'm Italian but I write you in English, it may be helpful to somebody else in the community. Surely it's my error but I can't install the integration of the Daikin residential controller on my HA. I followed the readme copy and pasting all the folder called daikin_residential and its content into config/custom_components folder, added the lines on the yaml file with user and pass for login to the daikin residential app but when I check configuration.yaml file system report an error because the daikin integration is not found. Do you know why?
Grazie !

Andrea
<img width="481" alt="Schermata 2021-11-05 alle 00 32 55" src="https://user-images.githubusercontent.com/92408005/140435483-1d631244-2417-4bba-8dae-c3b42f186951.png">

**Comment by rospogrigio** [comment]
Maybe you put the entire repository inside custom_components? You should just copy the folder daikin_residential/custom_components/daikin_residential inside custom_components. However, if you'd like to receive updates, you'd better install it using HACS: enter HACS -> Integrations, click on the 3 dots on the top right, select Custom repositories, add my repo's URL and select "Integration", then press "ADD": this will install the integration and will warn you when I publish new releases.
Also, you might want to try the Config flow instead o f the YAML editing: go to Configuration -> Integrations, press "Add integration" and select Daikin Residential, input your credentials there and the integration should install flawlessly.
Let me know how it goes, bye!

**Comment by genga91** [comment]
Don't know what I was making wrong, I added only the folder daikin_residental into custom_components. Anyway I clean the preview installation and added the GitHub folder like suggested, then into Integration I found the daikin residential controller and after few easy steps I found all my units! Working fine and it's even better than the official daikin ac integration, which didn't find some of my units (the ones with BRP069C4x Wifi board) and does not have power consumption monitoring. At this point I ask you how the consumption is monitored, is that daily consumption? Cause it doesn't seems to monitor it instantaneously. Many thanks for your integration btw, it's great!
